cmd/build.go

```diff
@@ -157,7 +171,53 @@ func newBuildCmd() *cobra.Command {
 				return err
 			}
 
-			api := apiNew(entry.APIURL, entry.Token)
+			if entry.UUID == "" {
+				fmt.Println("sd-local collects UUIDs for statistical surveys.")
+				fmt.Println("You can reset it later by removing the UA key from config.")
```

Comment:
This message explains how to reset the UUID, but once you reset the UUID, you will not be able to see this message unless you manually remove the UUID from the config file.

Reply:
What do you want me to do?

Reply:
I'm sorry.
I think it would be good to add an explanation in the help section of the config view command that it can be reset by changing the config file.

Reply:
Thanks. I tried the following:
```
Can see the below settings:
* Screwdriver.cd API URL
* Screwdriver.cd Store URL
* Screwdriver.cd Token
* Screwdriver.cd launcher version
* Screwdriver.cd UUID; Delete this key from config to reset UUID.
* Screwdriver.cd launcher image
```

However, few users see this message and try again. It also seemed to refer to something outside the context of a simple help statement. I'm sorry I couldn't fulfill your request.

Reply:
I see.
Thanks.